homeassistant/components/netatmo/climate.py

```diff
@@ -237,6 +238,10 @@ async def async_added_to_hass(self) -> None:
                 )
             )
 
+        registry = await self.hass.helpers.device_registry.async_get_registry()
```

Comment:
Avoid the use of `hass.helpers`, instead import async_get_regustry and pass the `hass` object into it. This helps us with things like typing.